Gemfile

```diff
@@ -49,7 +49,7 @@ group :active_record do
   platforms :ruby, :mswin, :mingw, :x64_mingw do
     gem 'mysql2', '>= 0.3.14'
     gem 'pg', '>= 1.0.0'
-    gem 'sqlite3', '~> 1.3'
+    gem 'sqlite3', '~> 2.2'
```

Comment:
Changing a Gemfile entry causes every files under `gemfiles/` to change, once you run `appraisal generate`. If your intention is to use sqlite3 ~> 2.2 only for Rails 8, you need to move this to Appraisal so other Rails versions can stay on sqlite 1.x.

Reply:
I've upgraded the sqlite version for all.

Reply:
You shouldn't. That's why builds for older Rails versions are failing.
https://github.com/railsadminteam/rails_admin/actions/runs/11931789920/job/33255380343?pr=3702

Reply:
@jewilmeer 
Please check, as this https://github.com/railsadminteam/rails_admin/pull/3702#discussion_r1851135751 is linked with @mshibuya's comment.